data/base-picking.yaml

```diff
@@ -66,30 +86,114 @@ picking:
-  - should not take long with your skills
-  - trap has the edge on you, but you've got a good shot at disarming
-  - You have some chance of being able to disarm
-  - odds are against you
+  traps:
+    acid: you notice a tiny hole right next to the lock
+    boomer: surrounded by a tight ring of fibrous cord
+    bouncer: Connected to the pin is a small shaft that runs downward into a shadow
+    curse: you notice a small glowing rune hidden inside the box near the lock
+    frog: you notice a lumpy green rune hidden inside the
+    reaper: crust-covered black scarab of some unidentifiable substance
+    crossbow: concealing the points of several wickedly barbed crossbow bolts
+    poisoncrossbow: concealing the points of several crossbow bolts glistening with moisture
+    concussion: you see a tiny metal tube just poking out of a small wad of brown clay
+    cyanide: tip of a dart and a slight smell of almonds
+    disease: you see what appears to be a small, swollen animal bladder
+    fireants: The bag twitches on occasion, leading you to believe the blade
+    fleas: is a small glass tube of milky-white opacity. Small black dots bounce inside
+    laughgas: a tiny glass tube filled with a black gaseous substance
+    mana: seal is covered in strange runes and a glass sphere is embedded within
+    mime: A tiny bronze face, Fae in appearance, grins ridiculously
+    poisongas: You notice a vial of lime green liquid just under the
+    lightning: Looking closely into the keyhole, you spy what appears to be a pulsating ball
+    napthasoak: Though it's hard to see, there also appears to be a liquid-filled bladder inside the notch
+    napthafire: A tiny striker is cleverly concealed under the lid, set to ignite a frighteningly large vial
+    poison: You notice a tiny needle with a greenish discoloration
+    nervepoison: You notice a tiny needle with a rust colored discoloration
+    scythe: you notice a glint of razor sharp steel hidden within a suspicious looking seam
+    shocker: You notice two silver studs right below the keyhole
+    shrapnel: keyhole is packed tightly with a powder around the insides of the lock
+    teleport: covered with a thin metal circle that has been lacquered with a shade
+    shadowling: you notice a small black crystal deep in the shadows of the
+    sleep: Two sets of six pinholes on either side of the
 
-  disarm_quick:
-  - trivially constructed gadget which you can take down any time
-  - This trap is a laughable matter
-  - An aged grandmother could defeat this trap in her sleep
+  disarmed_traps:
+    #acid:
+    boomer: separated harmlessly from their charge
+    #bouncer:
+    curse: You see a glowing rune pushed deep within the chest
+    frog: It seems far enough away from the lock to be harmless
+    crossbow: have been bent in such a way that they no longer will function
+    #concussion:
+    cyanide: the dart has been moved too far out of position for the mechanism to function
+    #disease:
```

Comment:
You notice a some type of animal bladder and a disconnected string near the lock.

Reply:
Awesome. Added.

Only six more types to go!